db/setup/deployment-mass-import.py

```diff
@@ -0,0 +1,528 @@
+# Returns the picture metadata object populated with the metadata
+def PictureSystemPopulating(
+    data,
+    date: date,
+    userID: str,
+    indexID: str,
+    format: str,
+    height: int,
+    width: int,
+    resolution: str,
+    source: str,
+    parent: str,
+):
+    """
+    Function to populate the Picture metadata file provided.
+
+    Parameters:
+    - data (str): The JSON data template of the Picture.
+    - date (date): The date of the upload
+    - userID (str): The UUID of the user
+    - indexID (str): The ID of the index
+    - format (str): The format of the picture
+    - height (int): The height of the picture
+    - width (int): The width of the picture
+    - resolution (str): The resolution of the picture
+    - source (str): The source of the picture
+    - parent (str): The parent of the picture
+
+    Returns:
+    - The Picture metadata object populated with the metadata.
+    """
+    data["info"]["userID"] = userID
+    data["info"]["uploadDate"] = date.strftime("%Y-%m-%d")
+    data["info"]["indexID"] = indexID
+    data["imageData"]["height"] = height
+    data["imageData"]["width"] = width
+    data["imageData"]["format"] = format
+    data["imageData"]["source"] = source
+    data["imageData"]["parent"] = parent
+    data["imageData"]["resolution"] = resolution
+
+    data["qualityCheck"]["imageChecksum"] = ""
+    data["qualityCheck"]["uploadCheck"] = True
+    data["qualityCheck"]["validData"] = True
+    data["qualityCheck"]["errorType"] = ""
+    data["qualityCheck"]["dataQualityScore"] = 1.0
+
+    return data
+
+
+def getImageProperties(path: str):
+    """
+    Function to retrieve an image's properties.
+
+    Parameters:
+    - path (str): The path to the image.
+
+    Returns:
+    - The image's width, height and format as a tuple.
+    """
+    with Image.open(path) as img:
+        width, height = img.size
+        img_format = img.format
+    return width, height, img_format
+
+
+# Function to open the system index metadata template file
+# Returns an empty system index metadata object
+def openIndexSystem():
+    """
+    Function to open the system index metadata template file.
+
+    Returns:
+    - An empty system index metadata object.
+    """
+    with open("index-template-system.json", "r") as file:
+        sysData = json.load(file)
+        # print(sysData)
+    return sysData
+
+
+# Function to open the system picture metadata template file
+# Returns an empty system picture metadata object
+def openPictureSystem():
+    """
+    Function to open the system picture metadata template file.
+
+    Returns:
+    - An empty system picture metadata object.
+    """
+    with open("picture-template-system.json", "r") as file:
+        sysData = json.load(file)
+        # print(sysData)
+    return sysData
+
+
+# Create .json from Index data model
+def createJsonIndex(index, name: str, output: str):
+    """
+    Create .json from index data model to the specified output.
```

Comment:
Duplicate comment, be careful, you don't need to comment on top of a function who have a docstring documentation

Reply:
fixed